src/Features/Core/Portable/PullMemberUp/QuickAction/ClassPullerWithQuickAction.cs

```diff
@@ -9,6 +9,24 @@ namespace Microsoft.CodeAnalysis.PullMemberUp.QuickAction
 {
     internal class ClassPullerWithQuickAction : AbstractMemberPullerWithQuickAction
     {
+        private static ClassPullerWithQuickAction s_puller;
+
+        internal static ClassPullerWithQuickAction GetInstance
```

Comment:
1. being lazy here saves us practically nothin.  these are cheap allocations.
2. calling a property 'GetInstance' isn't good.  Just make static readonly variables called 'Instance'.

Reply:
Got you